rpms/pulp-scl-python-billiard/pulp-scl-python-billiard.spec

```diff
@@ -0,0 +1,50 @@
+%{?scl:%scl_package python-billiard}
+%{!?scl:%global pkg_name %{name}}
+
+%global srcname billiard
+
+Name:		%{?scl_prefix}python-billiard
+Version:	3.3.0.23
+Release:	1%{?dist}
+Summary:	Multiprocessing Pool Extensions
+
+License:	BSD
+URL:		http://pypi.python.org/pypi/billiard
+Source0:	http://pypi.python.org/packages/source/b/%{srcname}/%{srcname}-%{version}.tar.gz
+
+BuildRequires:	%{?scl_prefix_python}python-devel
+BuildRequires:	%{?scl_prefix_python}python-setuptools
+BuildRequires:	%{?scl_prefix_python}python-nose
+%{?scl:BuildRequires:	%{scl}-build %{scl}-runtime}
+
+%{?scl:Requires:	%{scl}-runtime}
+
+BuildRoot:      %{_tmppath}/%{pkg_name}-%{version}-%{release}-root-%(%{__id_u} -n)
```

Comment:
I'm not sure if we need this. I'm not seeing this being referenced to in the build log. And according to https://fedoraproject.org/wiki/How_to_create_an_RPM_package

> BuildRoot: This is where files will be "installed" during the %install process (after the %build process). This is now redundant in Fedora and is only needed for EPEL5. By default, the build root is placed in "%{_topdir}/BUILDROOT/".